tasks/responsive_images.js

```diff
@@ -313,11 +313,22 @@ module.exports = function(grunt) {
   var processImage = function(srcPath, dstPath, sizeOptions, tally, callback) {
     var image = gfxEngine(srcPath);
 
-    image.identify(function(err, data) {
+    image.identify("%m:%T:%s\n", function(err, dataRaw) {
       if(err){
         handleImageErrors(err, sizeOptions.engine);
       }
 
+      var lastLineData = dataRaw.trim().split("\n").slice(-1)[0].split(":");
```

Comment:
`\n` should better be [`require('os').EOL`](https://nodejs.org/api/os.html#os_os_eol) to ensure platform compatibility.